apps/managedidentity/managedidentity_test.go

```diff
@@ -16,21 +19,49 @@ import (
 )
 
 const (
-	// test Resources
-	resource              = "https://demo.azure.com"
-	resourceDefaultSuffix = "https://demo.azure.com/.default"
-
-	token = "fakeToken"
+	// Test Resources
+	resource              = "https://management.azure.com"
+	resourceDefaultSuffix = "https://management.azure.com/.default"
+	token                 = "fakeToken"
+	azureArcTestEndpoint  = "http://127.0.0.1:40342/metadata/identity/oauth2/token"
```

Comment:
unused

Reply:
Used it across tests now and added azureArcImdsEndpoint